### PR TITLE
Minor corrections and elaborations to the Guide

### PR DESCRIPTION
**Pull request by fosskers**
This PR fixes some small things I noticed while reading, as well as expands several sections to reflect recent learning in the hopes that other users can avoid similar pitfalls.

As an aside, would you be opposed to line-wrapping in the `.mess` files? Using single long lines is somewhat diff-averse. It renders somewhat okay on Github, but in-editor it's not so great.

### CODE REVIEW

File: docs/harmony.mess [hunk 1]
```diff
@@ -10,9 +10,23 @@ First, you will want to inherit from ``org.shirakumo.fraf.trial.harmony:settings
...
 
-In order to play back a sound, just access its resource and play it back:
+:: common lisp
+(define-asset (my-pool my-sound) trial-harmony:sound
+  #p"sound.mp3")
+::
+
+If you want to use Harmony's environments system to do horizontal mixing, define the environment using the ``org.shirakumo.fraf.trial.harmony:environment`` asset.
+
+Before a sound can be played, it must be "staged". It is sufficient to associate ``stage`` with any entity present in the associated scene:
```
Comment (fosskers) [inline docs/harmony.mess:22]: > It is sufficient to associate...

Is this assumption correct?

**Comment by Shinmera** [comment]
I *am* opposed to line-wrapping text. It's text. Use something like olivetti-mode or whatever other editor tools to make the editor display suit your preferences. Manual line-wrapping forces those preferences onto everyone else.

File: docs/harmony.mess [hunk 2]
```diff
@@ -10,9 +10,23 @@ First, you will want to inherit from ``org.shirakumo.fraf.trial.harmony:settings
 
 See ``harmony:make-simple-server`` for more information on accepted initargs.
 
-Second, you will want to pre-load sound files. The ``trial-harmony`` system provides assets and resources to encapsulate the Harmony objects you need to manage. To import sounds and simple music tracks, use the ``org.shirakumo.fraf.trial.harmony:sound`` to define your assets. If you want to use Harmony's environments system to do horizontal mixing, define the environment using the ``org.shirakumo.fraf.trial.harmony:environment`` asset.
+Second, you will want to pre-load sound files. The ``trial-harmony`` system provides assets and resources to encapsulate the Harmony objects you need to manage. To import sounds and simple music tracks, use the ``org.shirakumo.fraf.trial.harmony:sound`` to define your assets:
 
-In order to play back a sound, just access its resource and play it back:
+:: common lisp
+(define-asset (my-pool my-sound) trial-harmony:sound
```
Comment (Shinmera) [inline docs/harmony.mess:16]: This is outdated, you should use trial's `audio-file` or `sound-bank` (for multiple effects that should be randomly picked between).

File: docs/harmony.mess (hunk 1 above)
Comment (Shinmera) [reply docs/harmony.mess:22]: It is correct, though there's also other options. You could subclass `scene` and use its stage method to do it, or you could use `preload` in `setup-scene`, or any number of ways. It's a question of how you want to organise the sounds.

**Comment by Shinmera** [comment]
I made the correction and merged it. Thanks for the updates!

**Pull request closed without merging**